util/chplenv/chpl_3p_hwloc_configs.py

```diff
@@ -14,5 +14,11 @@ def get_uniq_cfg_path():
                             chpl_locale_model.get())
 
 @memoize
-def get_link_args():
-    return third_party_utils.default_get_link_args('hwloc')
+def get_link_args(hwloc):
+    if hwloc == 'hwloc':
+        return third_party_utils.default_get_link_args('hwloc',
+                                                       ucp=get_uniq_cfg_path())
```

Comment:
Oh.. did we just not have it before, so we weren't actually reading any link args from the .la (we just got lucky that hwloc doesn't really have any dependencies)?

If that's the case can you update the PR message to note that